src/2d/shallow/multilayer/filval.f90

```diff
@@ -349,8 +352,8 @@ end subroutine filval
 
 subroutine dumpaux(aux,naux,mitot,mjtot)
    implicit none
-   real(kind=8) :: aux(naux,mitot,mjtot)
    integer :: naux,mitot,mjtot,i,j,iaux
+   real(kind=8) :: aux(naux,mitot,mjtot)
```

Comment:
Revert this as it does not actually add anything and we can then check versions easier in the future.